lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -938,11 +938,11 @@ DotOperandEncodingAttr::getElemsPerThread(ArrayRef<int64_t> shape,
     elemsPerThread[rank - 1] = (idx == 0) ? rep[2] * kWidth : rep[2];
     return elemsPerThread;
   } else if (auto mma = mlir::dyn_cast<NvidiaMmaEncodingAttr>(parent)) {
-    if (mma.isAmpere()) {
+    if (mma.isAmpere() || mma.isHopper()) {
```

Comment:
remove this line as well

Reply:
This wasn't removed, but it's probably fine. We can grep all these and clean them in a different PR.